src/cli/msig/addOpsKey.ts

```diff
@@ -0,0 +1,51 @@
+import { Name } from '@wharfkit/antelope';
+import { generateRandomKeyPair, getAccountInfo } from '../../sdk';
+import { ActionData, Authority } from '../../sdk/services/blockchain';
+
+export default async function addOpsNewKey() {
+    console.log('Creating new key\n');
+
+    // Generate a new key pair
+    const keyPair = generateRandomKeyPair();
```

Comment:
this is good but we need to be able to generate the private key first ( before running the CLI) so that we can save the private key. Then during the script we just need to provide the public key and it will add it to the blockchain.